assignments/objects.js

```diff
@@ -18,27 +18,70 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  "id": 01,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
```

Comment:
Interesting that you decided to go with JSON, having your `keys` and applicable values stringified. This is how all data goes back and forth to servers (at least those that use JSON so most).